monitor/BatteryMonitor.java

```diff
@@ -0,0 +1,163 @@
+package monitor;
+
+import properties.BatteryMessages;
+import vitals.Battery;
+import vitals.BatteryThresholds;
+import vitals.BatteryThresholds.BatteryState;
+
+public class BatteryMonitor {
+
+	static BatteryThresholds batteryThresholds = new BatteryThresholds();
+
+	static Boolean checkIfHigher(float valueToCheck, float threshold) {
+		if (valueToCheck > threshold) {
+			return true;
+		}
+		return false;
+	}
+
+	static Boolean checkIfLower(float valueToCheck, float threshold) {
+		if (valueToCheck < threshold) {
+			return true;
+		}
+		return false;
+	}
+
+	static Void checkForLowTemperatureBreach(Battery battery) {
+		if (checkIfLower(battery.temperature, batteryThresholds.batteryStateThresholds.get(BatteryState.TEMPERATURE)
+				.get("LOW_TEMPERATURE_BREACH_LIMIT"))) {
+			battery.statusReporter.printReport(BatteryMessages.getString("BatteryStatusChecker.LowTemperatureError")
+					+ BatteryMessages.getString("Battery.AbnormalValue") + battery.temperature);
+			battery.batteryStatus = false;
+		}
+		return null;
+	}
+
```

Comment:
function name begin with check and caller expecting  result but returns void 
Looks like Battery class is FAT for each check function....for example checkForLowTemperatureBreach  expects only values to determine breach but function knows every public interface of the battery class